src/System.Private.ServiceModel/tests/Common/Infrastructure/src/project.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "System.Runtime.InteropServices.RuntimeInformation" :  "4.0.0-beta-*",
     "System.ServiceModel.Duplex": "4.0.1-beta-*",
```

Comment:
Please double check -- if this package does not exist in NET Native, I believe this dependency could break.  If add #if code to make use of RuntimeInformation in CoreCLR only, would need to move the position of this dependency.